icicle/backend/cpu/include/ntt_tasks_manager.h

```diff
@@ -86,6 +86,7 @@ namespace ntt_cpu {
     uint32_t nof_pending_tasks = 0; // the current count of tasks that are pending execution
 
   private:
+    const uint32_t logn;                             // log of the NTT size
```

Comment:
I didn't fully understand the logn change.

Reply:
Discussed this in person